### PR TITLE
Add `formatter` attribute with `nixpkgs-fmt`

### PR DESCRIPTION
**Pull request by drupol**
This PR:

- [x] Add the `formatter` output
- [x] Reformat the code style with the default formatter `nixpkgs-fmt`
- [x] Add Github action to check the coding style

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -4,11 +4,21 @@
   inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
   inputs.flake-utils.url = "github:numtide/flake-utils";
 
-  outputs = { self, nixpkgs, flake-utils }:
+  outputs =
+    { self
+    , nixpkgs
+    , flake-utils
+    ,
```
Comment (figsoda) [inline flake.nix:11]: this doesn't look right, this line can be kept unchanged

**Review by figsoda — CHANGES_REQUESTED** [review]

File: flake.nix (hunk 1 above)
Comment (drupol) [reply flake.nix:11]: you're right, I did a mistake in the first commit. Going to fix this right now.

File: .github/workflows/nix-fmt-checks.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+      - name: Verify Changed files
+        uses: tj-actions/verify-changed-files@v13
+        id: verify_changed_files
+
+      - name: Files has changed
+        if: steps.verify_changed_files.outputs.files_changed == 'true'
+        run: |
+          echo "Repository has uncommitted changes, run nix fmt locally and commit the result."
+          exit 1
```
Comment (figsoda) [inline .github/workflows/nix-fmt-checks.yaml:27]:
```suggestion
        run: nix fmt . -- --check
```

Comment (drupol) [reply .github/workflows/nix-fmt-checks.yaml:27]: Nice ! Going to take care of that tomorrow, went to bed already.

Comment (drupol) [reply .github/workflows/nix-fmt-checks.yaml]: Fixed!

**Comment by Mic92** [comment]
Thanks!

**Pull request merged**